source/js/buyers-guide/index.js

```diff
@@ -1,5 +1,2 @@
-export {
-  bindWindowEventHandlers,
```

Comment:
Moving `bindWindowEventHandlers` because

<img width="928" alt="Screen Shot 2020-09-09 at 6 34 13 PM" src="https://user-images.githubusercontent.com/2896608/92777667-42ff0380-f355-11ea-86de-6602ce6c47fb.png">

Reply:
O_O